go/vt/sqlparser/ast.go

```diff
@@ -1276,51 +1264,38 @@ func (node *ColName) Equal(c *ColName) bool {
 	return node.Name.Equal(c.Name) && node.Qualifier.Equal(c.Qualifier)
 }
 
+// ValTuple represents a list of value expressions.
+// It can be RowTuple or ParenExpr.
+type ValTuple interface {
+	iRowTuple()
+	Expr
+}
+
+func (RowTuple) iRowTuple()  {}
+func (ParenExpr) iRowTuple() {}
```

Comment:
Yep, you're right. Expr already is a supertype of both ParenExpr and RowTupl. Integrated your patch, and removed the now unnecessary diffs of renaming ValTuple to RowTuple.

Gonna add some tests now